src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,185 @@
+import React, {Component, createRef} from 'react';
+import uniqId from 'uniqid';
+import {Button} from '../Button/Button';
+import './AddUserForm.scss';
+import ImgWarning from './../../assets/icon-attention-png-clip-art.png'
+
+const CN = 'user-form';
+
+class AddUserForm extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentError: '',
+            inputErrorName: '',
+            inputErrorLastName: '',
+            errorInputName: '',
+            errorInputLastName: ''
+        };
+    }
+
+    // todo: добавить state, в него добавить пропертю, которая будет отображать есть ли ошибка или нет.
+    //  допустим это будет строка currentError
+    //  по умолчанию она будет пустой
+    //  если произошла ошибка - в эту пропертю запишется строка, какая именно ошибка произошла
+
+    userNameRef = createRef();
+    userLastNameRef = createRef();
+
+    onInputChange = (e) => {
+
+        if (e.target.value) {
+            this.setState({
+                currentError: ''
+            })
+        }
+
+        if (this.userNameRef.current.value) {
+            this.setState({
+                inputErrorName: '',
+                errorInputName: ''
+            })
+        }
+
+        if (this.userLastNameRef.current.value) {
+            this.setState({
+                inputErrorLastName: '',
+                errorInputLastName: ''
+            })
+        }
```

Comment:
You are doing the same logic in both cases 
It would be less code if you will join conditions on line 38 and line 45 using operator ||